docs: add and translation tutorials/buildimage.md (#10)
RadxaOS-SDK/rsdk: docs/tutorials/usage.md

Reviewer: `4. Select `Yes` to start the build process.`后面缺了双空格换行

From there, you can use autocompletion to query the supported command list.

> Running `rsdk help` command will display its help text.

Reviewer: ```suggestion
Running a command with `--help` argument will display its help text.
```
这里its是指代上面自动补全命令，所以在这里说执行`rsdk help`是不会获得对应结果的。
先改成这样规定我们的命令格式，然后再给缺少的命令补上对应的帮助吧

> # Use TUI

Reviewer: ```suggestion
## Use TUI to run guided tasks
```

`rsdk` command is the common CLI entry point.

> When it was run without any argument, it will run `rsdk-tui` instead.

Reviewer: ```suggestion
When it is run without any argument, `rsdk-tui` will be run instead.
```

> 1. Enter TUI

Reviewer: ```suggestion
1. Enter TUI
```

```bash
rsdk
```

> 2. Select `Build system image`

Reviewer: ```suggestion
2. Select the task you want to use. You can use arrow key to navigate, use Enter key to select, and use Escape key to leave current window.
   In this example we will choose `Build system image`:
```

```bash
┌─────────────────┤ RSDK ├──────────────────┐
│ Please select a task:                     │
│                                           │
│            Build system image             │
│            =========                      │
│            About                          │
│                                           │
│         <Ok>             <Cancel>         │
│                                           │
└───────────────────────────────────────────┘
```

> 3. Select which product you want to compile an image for

Reviewer: ```suggestion
3. Select which product you want to build. You can use the space bar to select the product, and the chosen product will have asterisk symbol placed in the parentheses. **Enter key does not select the product when used alone!**
   In this example we will choose `rock-5b-6_1`:
```

```bash
┌─────────────────┤ RSDK ├──────────────────┐
│ Please select a product:                  │
│                                           │
│  ( ) radxa-e25                        ▒   │
│  (*) rock-5b-6_1                      ▒   │
│  ( ) rock-5b                          ▒   │
│                                           │
│         <Ok>             <Cancel>         │
│                                           │
└───────────────────────────────────────────┘
```

> 4. Select `Yes` to sure to build Radxa OS image

Reviewer: ```suggestion
4. Select `Yes` to start the build process. `rsdk-tui` will then run the associated CLI commands to complete the task:
```

From there, you can use autocompletion to query the supported command list.

Running a command with `--help` argument will display its help text.

## Use TUI to run guided tasks

`rsdk` command is the common CLI entry point.

When it is run without any argument, `rsdk-tui` will be run instead.

> 1. Enter TUI
> 
> ```bash
> rsdk
> ```

Reviewer: ```suggestion
1. Start TUI Wizard by running `rsdk` in the terminal.
```

> 2. Select the task you want to use. You can use arrow key to navigate, use Enter key to select, and use Escape key to leave current window.

Reviewer: ```suggestion
2. Select the task you want to run.
   You can use arrow key to navigate, use Enter key to select, and use Escape key to leave current window.
```
可能结尾要加两个空格

>    In this example we will choose `Build system image`:

Reviewer: ```suggestion
   In this example, we will choose `Build system image`:
```

```bash
┌─────────────────┤ RSDK ├──────────────────┐
│ Please select a task:                     │
│                                           │
│            Build system image             │
│            =========                      │
│            About                          │
│                                           │
│         <Ok>             <Cancel>         │
│                                           │
└───────────────────────────────────────────┘
```

> 3. Select which product you want to build. You can use the space bar to select the product, and the chosen product will have asterisk symbol placed in the parentheses. **Enter key does not select the product when used alone!**

Reviewer: ```suggestion
3. Select the product you want to build.
   You can use the space bar to select the product, and the chosen product will have asterisk symbol placed in the parentheses. **Enter key does not select the product when used alone!**
```

>    In this example we will choose `rock-5b-6_1`:

Reviewer: ```suggestion
   In this example, we will choose `rock-5b-6_1`:
```

```bash
┌─────────────────┤ RSDK ├──────────────────┐
│ Please select a product:                  │
│                                           │
│  ( ) radxa-e25                        ▒   │
│  (*) rock-5b-6_1                      ▒   │
│  ( ) rock-5b                          ▒   │
│                                           │
│         <Ok>             <Cancel>         │
│                                           │
└───────────────────────────────────────────┘
```

> 4. Select `Yes` to start the build process. `rsdk-tui` will then run the associated CLI commands to complete the task:

Reviewer: ```suggestion
4. Select `Yes` to start the build process.
   `rsdk-tui` will then run the associated CLI commands to complete the task:
```
